src/date-picker/date-picker.jsx

```diff
@@ -149,11 +149,14 @@ const DatePicker = React.createClass({
 
   getDefaultProps() {
     return {
-      formatDate: DateTime.format,
+      formatDate: (date) => {
```

Comment:
I wonder how this shows up in the docs O.o any chance we can get the function out into a private top level function? And name it: `ISO8601CompliantFormatter` so this would show up on the docs? Seems right for documentation :sweat_smile: